data/language/en-GB.yml

```diff
@@ -2409,3 +2409,9 @@ strings:
   2354: "{COLOUR WINDOW_2}Maximum river width:"
   2355: "{COLOUR WINDOW_2}Riverbank width:"
   2356: "{COLOUR WINDOW_2}Meander rate:"
+  2357: "{COLOUR WINDOW_2}Sort music by:"
+  2358: "Original order"
+  2359: "Alphabetical order"
+  2360: "Reverse alphabetical order"
```

Comment:
I think `Alphabetical (reversed)` would be more consistent with the `Era (reversed)` below. I also think you don't need to write `order` on the first 3 items. Also I think `Year` sounds more natural than `Era`. I would do 

```
- Original
- Alphabetical
- Alphabetical (reversed)
- Year
- Year (reversed)
```